types/react-redux/index.d.ts

```diff
@@ -313,18 +316,17 @@ export interface ConnectOptions {
     withRef?: boolean
 }
 
-export interface ProviderProps {
+export interface ProviderProps<A extends Action> {
```

Comment:
Since this did not have a type parameter before, will this break existing users? It might be advisable to add a default for the type parameter.